cockatrice/src/window_main.cpp

```diff
@@ -1205,7 +1227,7 @@ int MainWindow::getNextCustomSetPrefix(QDir dataDir)
 
 void MainWindow::actManageSets()
 {
-    auto *w = new WndSets;
+    w = new WndSets;
```

Comment:
I think a one-character variable name was okay in this local scope, but since now it's made into a data member of a quite large class it probably should get a more expressive name.

Reply:
Addressed